aten/src/ATen/core/library.cpp

```diff
@@ -218,6 +218,30 @@ Library& Library::_impl(const char* name_str, CppFunction&& f) & {
   );
   return *this;
 }
+
+c10::OperatorHandle Library::_resolve(const char* name_str) const {
+  auto name = torch::jit::parseName(name_str);
```

Comment:
nit -- can create a helper that does the check on namespace as the same logic is repeated in impl

Reply:
yep, i'll do it after passing ci